style.css

```diff
@@ -0,0 +1,447 @@
+body {
+	font-family: 'Gotham Pro';
+	font-size: 14px;
+	max-width: 1030px;
+	min-width: 320px;
+	margin: 0 auto;
+	padding: 32.46px 0;
+}
+
+a {
+	text-decoration: none;
+	color: #000000;
+	text-transform: uppercase;
+}
+
+a:hover {
+	opacity: .5;
+}
+
+/*<header class='header'>*/
+.header {
+	color: #000000;
+	font-size: 12px;
+	font-weight: bold;
+	display: flex;
+	justify-content: space-between;
+	margin: 0 0 28.5px 0;
+}
+
+.header__conteiner {
+	flex: 0 1 50%;
+	display: flex;
+	align-items: center;
+}
+
+.header__img {	
+	height: 55px;
+	margin: 0 45px 0 0;
+}
+.header__logo {}
+.header__logo-mobile {display: none;}
+
+nav a, .header__link {
+	margin-right: 35px;
+	line-height: 11px;
+}
+
+.header__conteiner_right {justify-content: flex-end;}
+
+.search {
+	border-bottom: 1px solid #000000;
+	margin-right: 35px;
+}
+
+.search__form {width: 164px;}
+
+.header__link {vertical-align: middle;}
+
+.header__number::after {
+	content: counter(number);
+	counter-increment: number 1;
+	background: #000000;
+	display: inline-block;
+	width: 15px;
+	height: 15px;
+	color: white;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)